tests/prague/eip7002_el_triggerable_withdrawals/test_modified_withdrawal_contract.py

```diff
@@ -0,0 +1,212 @@
+"""
+abstract: Tests [EIP-7002: Execution layer triggerable withdrawals](https://eips.ethereum.org/EIPS/eip-7002)
+    Test execution layer triggered exits [EIP-7002: Execution layer triggerable withdrawals](https://eips.ethereum.org/EIPS/eip-7002).
+
+"""  # noqa: E501
+
+from typing import List
+
+import pytest
+
+from ethereum_test_tools import (
+    Account,
+    Alloc,
+    Block,
+    BlockchainTestFiller,
+    Bytecode,
+    Transaction,
+)
+from ethereum_test_tools import Macros as Om
+from ethereum_test_tools import Opcodes as Op
+from ethereum_test_types import Requests
+
+from .helpers import (
+    WithdrawalRequest,
+    # WithdrawalRequestBase,
+    # WithdrawalRequestContract,
+    WithdrawalRequestTransaction,
+)
+from .spec import Spec as Spec_EIP7002
+from .spec import ref_spec_7002
+
+REFERENCE_SPEC_GIT_PATH: str = ref_spec_7002.git_path
+REFERENCE_SPEC_VERSION: str = ref_spec_7002.version
+
+
```

Comment:
```suggestion

pytestmark = pytest.mark.valid_from("Prague")


```
Need to limit these tests to only run on Prague and after.